src/agent_factory/start.sh

```diff
@@ -7,9 +8,20 @@
 : "${PORT:=8080}"
 : "${LOG_LEVEL:=info}"
 : "${CHAT:=1}"
+: "${MAX_TURNS:=20}"
```

Comment:
Should we match the value in the `Makefile` and `Dockerfile` (40)?

```suggestion
: "${MAX_TURNS:=40}"
```